flexibee-core/src/main/java/com/adleritech/flexibee/core/api/FlexibeeClient.java

```diff
@@ -23,33 +30,20 @@
 import retrofit2.http.Path;
 import retrofit2.http.Query;
 
-import javax.net.ssl.HostnameVerifier;
-import java.io.IOException;
-import java.io.StringWriter;
-import java.security.KeyStore;
-
 public class FlexibeeClient {
 
-    private static final String API_BASE_URL = "https://demo.flexibee.eu:5434";
-
     private final String company;
 
-    private final Serializer xmlPrinter = Factory.persister();
-
     @Getter
     private final Api client;
 
-    public FlexibeeClient(String username, String password, String company) {
-        this(username, password, company, API_BASE_URL);
-    }
+    private final Converter<ResponseBody, WinstromResponse> errorConverter;
```

Comment:
Proc je tohle `errorConverter`? Konvertuje na obecnou WinstromResponse.

Reply:
Protože se používá jen pro error response. Ale v principu je pravda že je to prachobyčejný converter na WinstromResponse